ilc/client/errors/index.js

```diff
@@ -0,0 +1,7 @@
+import extendError from '@namecheap/error-extender';
+
+const errors = {};
+
+errors.CorsError = extendError('CorsError');
+
+export default Object.freeze(errors);
```

Comment:
why default export and object.freeze ?

Reply:
default export because it should be list of common client errors.
Object.freeze - to prevent list modification